scripts/cli-test.sh

```diff
@@ -47,8 +47,6 @@ $FM_MINT_CLIENT fetch
 # peg out
 PEG_OUT_ADDR="$($FM_BTC_CLIENT getnewaddress)"
 $FM_MINT_CLIENT peg-out $PEG_OUT_ADDR 500
-sleep 5 # FIXME wait for tx to be included
-await_fedimint_block_sync
 until [ "$($FM_BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR 0)" == "0.00000500" ]; do
   sleep $POLL_INTERVAL
```

Comment:
Not something to change in this PR, but IMO we should always have `&>2 echo "Waiting for xyz..." before sleep in these loops to have a visual clue about what is taking how much etc.